### PR TITLE
Readme Update about the Deployment of the App

### PR DESCRIPTION
**Pull request by CledersonE**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by simoneb** [comment]
The 44 commits seem a bit out of place

File: README.md [hunk 1]
```diff
@@ -61,5 +61,13 @@ Then this mechanism can do little to no harm, in the worst case merging a Depend
...
-- Configure the secrets for: `GCP_PROJECT_ID`, `GCP_CLOUDRUN_SERVICE_NAME`, `GCP_CLOUDRUN_SERVICE_REGION`, `GCP_SA_KEY`, `API_ID` and `PRIVATE_KEY`
-- After these steps above are configured, run the workflow
+- Clone this repo to your GitHub account
+- In the `Settings` of your GitHub repo, go to `Secrets` and create the `New repository secret` with the names and values bellow:
+    - `GCP_PROJECT_ID`: The [project ID](https://support.google.com/googleapi/answer/7014113?hl=en) of your GCP Account
+    - `GCP_CLOUDRUN_SERVICE_NAME`: The name of the cloud run service, you can select any name that you prefer
+    - `GCP_CLOUDRUN_SERVICE_REGION`: The [region](https://cloud.google.com/compute/docs/regions-zones) in the GCP that you want to create the cloud run service
+    - `GCP_SA_KEY`: The key that you created for your service account with the permissions to deploy the app
+    - `API_ID`: The ID of the api to run the dependabot-merge-action-app
```
Comment (mcollina) [inline README.md:70]: is this Github?

File: README.md [hunk 2]
```diff
@@ -61,5 +61,13 @@ Then this mechanism can do little to no harm, in the worst case merging a Depend
...
+    - `PRIVATE_KEY`: The private key to run the dependabot-merge-action-app
```
Comment (mcollina) [inline README.md:71]: is this Github?

**Review by mcollina — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -61,5 +61,13 @@ Then this mechanism can do little to no harm, in the worst case merging a Depend
...
-- Configure the secrets for: `GCP_PROJECT_ID`, `GCP_CLOUDRUN_SERVICE_NAME`, `GCP_CLOUDRUN_SERVICE_REGION`, `GCP_SA_KEY`, `API_ID` and `PRIVATE_KEY`
-- After these steps above are configured, run the workflow
+- Clone this repo to your GitHub account
+- In the `Settings` of your GitHub repo, go to `Secrets` and create the `New repository secret` with the names and values bellow:
+    - `GCP_PROJECT_ID`: The [project ID](https://support.google.com/googleapi/answer/7014113?hl=en) of your GCP Account
+    - `GCP_CLOUDRUN_SERVICE_NAME`: The name of the cloud run service, you can select any name that you prefer
+    - `GCP_CLOUDRUN_SERVICE_REGION`: The [region](https://cloud.google.com/compute/docs/regions-zones) in the GCP that you want to create the cloud run service
+    - `GCP_SA_KEY`: The key that you created for your service account with the permissions to deploy the app
```
Comment (simoneb) [inline README.md:69]:
```suggestion
    - `GCP_SA_KEY`: The key that you created for your service account with the permissions to deploy the app. This is a JSON object and should be used as-is.
```

File: README.md (hunk 1 above)
Comment (simoneb) [inline README.md:70]:
```suggestion
    - `APP_ID`: The ID of the GitHub App. You can get this from the GitHub app settings. The default app is [here](https://github.com/apps/dependabot-merge-action)
```

File: README.md (hunk 2 above)
Comment (simoneb) [inline README.md:71]:
```suggestion
    - `PRIVATE_KEY`: The private key of the GitHub App. You can get this from the GitHub app settings. The default app is [here](https://github.com/apps/dependabot-merge-action)
```

**Review by simoneb — APPROVED** [review]

**Comment by mcollina** [comment]
This needs a rebase

**Pull request merged**